docs/source/image_load.mdx

```diff
@@ -72,6 +72,14 @@ Load your dataset by specifying `imagefolder` and the directory of your dataset
 {"image": <PIL.PngImagePlugin.PngImageFile image mode=RGBA size=1200x215 at 0x15E8DAD30>, "label": 1}
 ```
 
+If your directory contains multiple splits, use the `data_files` argument to load the images associated with each split:
+
+```py
+>>> from datasets import load_dataset
+
+>>> dataset = load_dataset("imagefolder", data_files={"train":["train/dog/golden_retriever.png", ...], "test": ["test/dog/german_shepherd.png", ...)
+```
```

Comment:
You can put this below "Load remote datasets from their URLs with the `data_files` parameter:"

Reply:
no need for this paragraph here too, see my comment for audio_load.mdx :)